Habinezajanvier/Capstone
# #38 display admin profile

#### What does this PR do?
Displaying admin profiles
#### Description of Task to be completed?
- Getting user Infos from firebase
- Displaying Infos on page
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/Habinezajanvier/my-brand-project.git`
- Switch the directory `cd my-brand-project`
- Checkout the branch `git checkout ft-display-admin-profile-#38`
- Using your preferred browser start `index.html` found in `/admin/` folder
- Click on **Acount** link found in left-side nav-bar
> You should see Your Infos of profile photo, name and email
#### Any background context you want to provide?
- If the page redirects you to login, use **`admin@email.com`** and **admin123** as credentials
#### What are the relevant pivotal tracker stories?
Click [here](https://trello.com/c/G43qSvXM/38-display-admin-account) for relevant trello story
